Bio/Phenotype/PhenMicro.py

```diff
@@ -0,0 +1,1101 @@
+                else:             
+                    plateID = _platesPrefix + '%02d'%int(pID)
+                    
+            plate.id = plateID
+                
+        elif _strainType in line[0].strip():
+            if plate is None:
+                continue
+            qualifiers[_csvData][_strainType] = line[1].strip()
+        
+        elif _sample in line[0].strip():
+            if plate is None:
+                continue
+            qualifiers[_csvData][_sample] = line[1].strip()
+        
+        elif _strainNumber in line[0].strip():
+            if plate is None:
+                continue
+            qualifiers[_csvData][_strainNumber] = line[1].strip()
+        
+        elif _strainName in line[0].strip():
+            if plate is None:
+                continue
+            qualifiers[_csvData][_strainName] = line[1].strip()
+        
+        elif _other in line[0].strip():
+            if plate is None:
+                continue
+            qualifiers[_csvData][_other] = line[1].strip()
+            
+        elif _file in line[0].strip():
+            if plate is None:
+                continue
+            qualifiers[_csvData][_file] = line[1].strip()
+            
+        elif _position in line[0].strip():
+            if plate is None:
+                continue
+            qualifiers[_csvData][_position] = line[1].strip()
+            
+        elif _other in line[0].strip():
+            if plate is None:
+                continue
+            qualifiers[_csvData][_setupTime] = line[1].strip()
+        
+        elif _hour in line[0].strip():
+            if plate is None:
+                continue
+            data = True
+            for i in range(1, len(line)):
+                x = line[i]
+                if x == '':
+                    continue
+                wells[x.strip()] = {}
+                idx[i] = x.strip()
+                
+        elif data:
+            if plate is None:
+                continue
+            
+            # Workaround for bad-formatted files
+            try:
+                float(line[0])
+            except:
+                continue
+            
+            time = float(line[0])
+                                
+            for i in range(1, len(line)):
+                x = line[i]
+
+                try:
+                    signal = float(x)
+                except ValueError:
+                    continue
+                    
+                well = idx[i]
+                wells[well][time] = signal
+    
+    if plate is not None:
+        qualifiers[_csvData][_datafile] = line[1].strip()
+        plate = PlateRecord(plate.id)
+        for k, v in wells.items():
+            plate[k] = WellRecord(k, plate, v)
+        plate.qualifiers = qualifiers
+        yield plate
+
+def _toOPM(plate):
+    """Helper function to transform a PlateRecord object into a dictionary
+    """
+    d={}
+    
+    for k, v in plate.qualifiers.items():
+        d[k] = v
```

Comment:
Probably you can do that in one go, guessing  ``d = dict(plate.qualifiers.items())`` would work?